packages/typespec-python/src/emitter.ts

```diff
@@ -223,15 +223,28 @@ function getEntityType(context: DpgContext, entity: ModelProperty): any {
 function getType(context: DpgContext, type: EmitterType): any {
     // don't cache simple type(string, int, etc) since decorators may change the result
     const program = context.program;
-    const enableCache = !isSimpleType(context, type);
+    // object, {}, Model{} all will be treated as empty model
+    const isEmptyModel =
```

Comment:
nit: would be better to pull this out into a separate function for readability, i.e. `function isEmptyModel`

Reply:
Refined.